docs/proposals/202105256-cluster-class-and-managed-topologies.md

```diff
@@ -702,6 +771,11 @@ intentionally use resources without patches and variables to focus on the simple
           apiVersion: controlplane.cluster.x-k8s.io/v1alpha4
           kind: KubeadmControlPlaneTemplate
           name: vsphere-prod-cluster-template-kcp
+        # This will create a health check for each deployment created with the "linux-worker" MachineDeploymentClass
```

Comment:
Let's be explicit with the resource name:
```yaml
# This will create a MachineHealthCheck for "vsphere-prod-cluster-template-kcp" KubeadmControlPlane machines.
```